Tests/Functional/Utilities.swift

```diff
@@ -155,6 +155,44 @@ func executeSwiftBuild(_ chdir: String, configuration: Configuration = .Debug, p
     }
 }
 
+func swiftTestPath() -> String {
+    #if os(OSX)
+        for bundle in NSBundle.allBundles() where bundle.bundlePath.hasSuffix(".xctest") {
```

Comment:
saved the kitten in https://github.com/apple/swift-package-manager/pull/266/commits/852af2e1c2ff46dbb00d01fb413e04ad7019fe63